### DISCUSSION TITLE
Night vision goggles

### DISCUSSION THREAD

**Discussion by Hazmat616** [FakeFishGames/Barotrauma#12407] [Ideas and suggestions]
Goggle's similar to thermal goggles that let you see in dark areas but makes everything black and white or green and blinds or stuns you if you encounter bright lights. This would help you in dark environments without attracting extra attention by using a flashlight.

**Comment by jigardi** [comment]
Lots of fun potential with this one, have my upvote!

**Reply by SomeRandomNoobKekeke** [reply]
https://steamcommunity.com/sharedfiles/filedetails/?id=3294574390
